typography/main.scss

```diff
@@ -1,26 +1,25 @@
 @import 'o-fonts/main';
 @import 'o-typography/main';
 
-//BREAKING CHANGE: @include nLinksTopic -> @include oTypographyTopic
-@mixin nLinksTopic () {
-	@throw 'The nLinksTopic mixin has been removed. Please use o-typography’s oTypographyTopic mixin instead.';
-}
-
 @mixin nUiTypography {
-	@include oFontsInclude(FinancierDisplayWeb);
-	@include oFontsInclude(FinancierDisplayWeb, $weight: bold);
-	@include oFontsInclude(MetricWeb);
-	@include oFontsInclude(MetricWeb, $weight: semibold);
-	@include oFontsInclude(MetricWeb, $weight: medium);
+	@include oFonts($opts: (
+		'financier-display': (
+			('weight': 'regular', 'style': 'normal'),
+			('weight': 'bold', 'style': 'normal'),
+		),
+		'metric': (
+			('weight': 'regular', 'style': 'normal'),
+			('weight': 'semibold', 'style': 'normal'),
+			('weight': 'medium', 'style': 'normal'),
+		)
+	));
 
 	html {
-		//TODO - move elsewhere??
-		@include oColorsFor(page);
```

Comment:
This used to set background colours to pink. Did this get moved somewhere else?

Reply:
It's always been set by both `n-ui` and Page Kit so this was a duplicate!

https://github.com/Financial-Times/n-ui/blob/master/browser/layout/main.scss#L10-L13

https://github.com/Financial-Times/dotcom-page-kit/blob/master/packages/dotcom-ui-shell/src/components/Shell.tsx#L51-L57